packages/next/lib/typescript/writeConfigurationDefaults.ts

```diff
@@ -197,9 +197,17 @@ export async function writeConfigurationDefaults(
     chalk.green(
```

Comment:
@Pearcekieser  Something like this and I think it aligns with Tim's preferred solution.

```ts
 console.log(
    chalk.green(
      `We detected TypeScript in your project and reconfigured your ${chalk.bold(
        'tsconfig.json'
      )} file for you.${
        strictWasPreviouslySet
          ? ''
          : ` Strict-mode is set to ${chalk.bold('false')} by default.`
      }`
    ) + '\n'
  )
```